src/Draggable/Draggable.js

```diff
@@ -395,6 +395,7 @@ export default class Draggable {
     if (!isDragEvent(originalEvent) && !mirrorCreateEvent.canceled()) {
       const appendableContainer = this[getAppendableContainer]({source: this.originalSource});
       this.mirror = this.source.cloneNode(true);
+      this.mirror.style.display = 'none';
```

Comment:
Could you move this line to `Mirror.js`, inside the `onMirrorCreated` method? That way the logic for showing hiding is in the same module. We are also currently working on moving all mirror related logic from `Draggable` to `Mirror`